services/movie.py

```diff
@@ -0,0 +1,31 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> list[Movie]:
+    if not genres_ids and not actors_ids:
+        return Movie.objects.all()
+    elif genres_ids and actors_ids:
+        return Movie.objects.filter(genres__id__in=genres_ids,
+                                    actors__id__in=actors_ids)

```

Comment:
When filtering by both genres and actors, the current implementation uses `filter` with both conditions, which results in an 'AND' operation. This means it will only return movies that have at least one of the specified genres AND at least one of the specified actors. If the intention is to return movies that match either condition, consider using `Q` objects to combine the filters with an 'OR' operation.